polygon-validator-signer-balance-bot/src/findings.ts

```diff
@@ -2,13 +2,13 @@ import { Finding, FindingSeverity, FindingType } from "forta-agent";
 import { MINIMUM_THRESHOLD } from "./constants";
 
 export const createUnderThresholdFinding = (balance: string) => {
+  console.log(MINIMUM_THRESHOLD.toString());
   return Finding.fromObject({
     name: "Account balance below threshold!",
-    description: `Account balance (${balance.toString()}) below threshold (${MINIMUM_THRESHOLD})`,
-    alertId: "POLY01",
+    description: `Account balance (${balance}) below threshold (${MINIMUM_THRESHOLD.toString()})`,
```

Comment:
Right now, `balance` and `MINUMUM_THRESHOLD` are in wei. They can be formatted, only so that they look better. You could also add the word "ETH" after both.